src/gui/SetupDialog.cpp

```diff
@@ -839,8 +839,18 @@ SetupDialog::SetupDialog( ConfigTabs _tab_to_open ) :
 		ConfigManager::inst()->setValue(
 					"mixer", "audiodev", audioDevName );
 	}
-	m_audioInterfaces->
-		setCurrentIndex( m_audioInterfaces->findText( audioDevName ) );
+
+	int index = m_audioInterfaces->findText(audioDevName);
+	//check index, during debugging (dis/en)abling audio devices this can cause a crash (index -1)
+	//TODO: output error msg? 
+	if (index < 0)
```

Comment:
I think you can merge the check into line 836 because it does the same thing inside `if` clauses.